astroquery/heasarc/core.py

```diff
@@ -171,7 +171,7 @@ def query_region_async(self, position: Union[coordinates.SkyCoord, str],
         # Generate the request
```

Comment:
you may want to leave a one-liner comment as a reminder why formatting is applied here, otherwise it's likely someone would wonder a few years down the line 😅

Reply:
Sure, done. I did not manage in one line, only two, but if you have better ideas - please suggest!

Reply:
multi-liner is fine, too, I just meant that it can be just a very short reminder :)